transform/mattermost-analytics/models/marts/product/_product__models.yml

```diff
@@ -355,4 +355,48 @@ models:
         description: |
           Total number of active users for the past 24 hours prior to the timestamp of the event.
           Bots and deleted users are excluded. Reported by mattermost server. If there is no 
-          information about the specific date (i.e. server inactive or server hasn't reported telemetry), then it's 0.
\ No newline at end of file
+          information about the specific date (i.e. server inactive or server hasn't reported telemetry), then it's 0.
+
+  - name: dim_installation_summary
+    description: | 
+      Summary of each installation based on the telemetry data. Contains data for servers with an existing 
+      installation id only.
+
+    columns:
+      - name: installation_id
+        description: The id of the installation.
+        tests:
+          - not_null
+          - unique
+      - name: installation_id
```

Comment:
duplicate?

Reply:
Hah, seems I accidentally added that using Github Copilot. Nice catch!

Reply:
Next step - adding comments (like this one) to PRs = )